aten/src/ATen/native/transformers/sdp_utils_cpp.h

```diff
@@ -274,6 +274,17 @@ inline bool check_for_attn_mask(sdp_params const& params, bool debug) {
   return true;
 }
 
+// TODO(eqy): remove this once support is added
```

Comment:
https://docs.nvidia.com/deeplearning/cudnn/archives/cudnn-894/developer-guide/index.html#fused-multi-head-att-fprop it looks like fused attention support padding mask, is this version currently not planning to support mask?

Reply:
We have not added it yet, the next feature in-flight is attn_bias: https://github.com/pytorch/pytorch/pull/130482

Reply:
Got it, thank you for your response. We would like to know if you have any plans to support fused attention with padding mask in the future? This feature is very important for our work, and it would be great if we could directly reuse your work.